client/src/test/java/com/influxdb/client/WriteApiTest.java

```diff
@@ -646,8 +648,8 @@ void retryWithRetryAfter() throws InterruptedException {
 
         retriableListener.awaitCount(1);
         Assertions.assertThat(retriableListener.getValue().getThrowable())
-                .isInstanceOf(HttpException.class)
-                .hasMessage("HTTP 429 Client Error");
+                .isInstanceOf(InfluxException.class)
+                .hasMessage("token is temporarily over quota");
```

Comment:
This test logs to console:
```
Oct 15, 2020 10:20:15 AM com.influxdb.client.write.events.WriteRetriableErrorEvent logEvent
WARNING: The retriable error occurred during writing of data. Reason: {0}. Retry in: 5,000ms.
```
Reason: {0} is not expanded.

Reply:
Done